cyber-security/components/csls-splunk-broker/pkg/csls/handler.go

```diff
@@ -61,8 +65,9 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprintln(w, "OK")
 }
 
-// process decodes a cloudfoundry format syslog line and forward it to kinesis stream
-func (h *Handler) process(r io.Reader, code string) error {
+// process decodes a cloudfoundry format syslog line and forward it to kinesis
+// stream with the service instance GUID as the log group name
+func (h *Handler) process(r io.Reader, code string, serviceInstanceGUID uuid.UUID) error {
```

Comment:
yeah it's bit generic as the function only really exists so that error handling can be done at the call site.

we could go full descritive with something like `transformAndForwardLogEvent` ?

Reply:
That's definitely an improvement. I'm happy with `transformAndForwardLogEvent`